pkg/gke/testsdkconfig/failure/active_config

```diff
@@ -0,0 +1 @@
+test
```

Comment:
better to write these simple files in the tests they are being used: it is more maintainable to define the files there, if not you don't know which file is related to which test.

Reply:
I intended to do that way, but I changed my mind because: the project contains a very small number of files and keep those separate files reduces the coding cost as well as increases visibility: Each folder test contains set of files for that test, not a single file. I am also happy to embed those into code. So I vote 0 for this (in the sense of -1, 0, 1 in voting) :p. Right now I just land the PR, we can do it in a separate PR if we want :p